gradle/content.md

```diff
@@ -0,0 +1,22 @@
+# What is Gradle?
+
+[Gradle](https://gradle.org/) is a build tool with a focus on build automation and support for multi-language development. If you are building, testing, publishing, and deploying software on any platform, Gradle offers a flexible model that can support the entire development lifecycle from compiling and packaging code to publishing web sites. Gradle has been designed to support build automation across multiple languages and platforms including Java, Scala, Android, C/C++, and Groovy, and is closely integrated with development tools and continuous integration servers including Eclipse, IntelliJ, and Jenkins.
+
+%%LOGO%%
+
+# How to use this image
+
+## Building a Gradle project
+
+Run this from the directory of the Gradle project you want to build.
+
+`docker run --rm -v "$PWD":/project -w /project --name gradle gradle:alpine gradle <gradle-task>`
+
+## Reusing the Gradle cache
+
+The local Gradle cache can be reused across containers by creating a volume and mounting it in */root/.gradle*.
+
+```console
+docker volume create --name gradle-cache
+docker run -it -v gradle-cache:/root/.gradle gradle:latest gradle build
```

Comment:
This is technically different now given your latest update to the image, right? (adding a user and a volume on that user's `.gradle` directory)